hardware.inc

```diff
@@ -361,10 +361,10 @@ DEF rAUD4GO EQU rNR44
 ; -- AUDVOL/NR50 ($FF24)
 ; -- Channel control / ON-OFF / Volume (R/W)
 ; --
-; -- Bit 7   - Vin->SO2 ON/OFF (Vin??)
-; -- Bit 6-4 - SO2 output level (volume) (# 0-7)
-; -- Bit 3   - Vin->SO1 ON/OFF (Vin??)
-; -- Bit 2-0 - SO1 output level (volume) (# 0-7)
+; -- Bit 7   - Vin->SO2 ON/OFF (left)
+; -- Bit 6-4 - SO2 output level (left volume) (# 0-7)

```

Comment:
```suggestion
; -- Bit 6-4 - SO2 output level (left channel) (# 0-7)

```

Reply:
This is confusing with the four *input* "channels", so "speaker", "terminal", or "output" is better; though the latter is redundant with the left part... do we need the parens at all?

Reply:
I agree with speaker and output